packages/admin-panel/src/VizBuilderApp/vizBuilderConfigStore.js

```diff
@@ -124,28 +106,22 @@ function configReducer(state, action) {
 const useConfigStore = () => {
   const [state, dispatch] = useReducer(configReducer, initialConfigState);
 
-  const setLocation = useCallback(value => dispatch({ type: SET_LOCATION, value }), []);
```

Comment:
No point calling `useCallback` with an empty dependency array. I removed such usages to keep things simple (and ""optimised"")